src/content-helper/common/components/beta-badge.tsx

```diff
@@ -0,0 +1,44 @@
+import '../css/common.scss';
+
+/**
+ * Properties for the BetaBadge component.
+ */
+type BetaBadgeProps = {
+	/**
+	 * The text to display in the badge. If not provided, defaults to 'Beta'.
+	 */
+	text?: string;
+
+	/**
+	 * The color of the badge. If not provided, defaults to 'var(--parsely-green)'.
+	 */
+	color?: string;
+
+	/**
+	 * The size of the badge text. If not provided, defaults to '0.75rem'.
+	 */
+	size?: string;
+};
+
+/**
+ * The BetaBadge component displays a badge with the specified text, color, and size.
+ *
+ * @param {BetaBadgeProps} props - The properties for the BetaBadge component.
+ * @return {JSX.Element} The BetaBadge component.
```

Comment:
```suggestion

 * @return {JSX.Element} The BetaBadge component.
```

The tricky part here is that for JS we have a blank line before `@param` and `@return`, and for PHP we don't. This is a detail to remember, as unfortunately this is how things are in WP inline documentation standards. Otherwise, we can discuss having a common approach for everything.